### PR TITLE
Update teams-recording-policy.md

### PR DESCRIPTION
**Pull request by 2kiran**

### CODE REVIEW

File: Teams/teams-recording-policy.md [hunk 1]
```diff
@@ -320,4 +320,4 @@ This list will be updated as more partners join and meet the certification crite
 
 ## Next steps
 
-If you are a vendor seeking to join the certification program, send mail to [Teamscategorypartner@microsoft.com](mailto:Teamscategorypartner@microsoft.com).
+If you are a vendor seeking to join the certification program, please fill out the form [http://aka.ms/CallingPlatformIntake](http://aka.ms/CallingPlatformIntake) as the next step. If you need to provide additional context and details, send a mail to [Teamscategorypartner@microsoft.com](mailto:Teamscategorypartner@microsoft.com).
```
Comment (JohanFreelancer9) [inline Teams/teams-recording-policy.md:323]:
```suggestion
If you are a vendor seeking to join the certification program, fill out [this form](http://aka.ms/CallingPlatformIntake) as the next step. If you need to provide additional context and details, send a mail to [Teamscategorypartner@microsoft.com](mailto:Teamscategorypartner@microsoft.com).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @2kiran, kindly commit the suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@cabailey PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by cabailey** [comment]
@2kiran Apologies for missing this previously - because your GitHub profile is still not updated.  To ensure your PRs are merged efficiently as the PM owner, please update your profile.

**Pull request merged**